Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uxml

```diff
@@ -0,0 +1,46 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" noNamespaceSchemaLocation="../../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <ui:Template name="InfoLinks" src="project://database/Packages/com.amazonaws.gamelift/Editor/Resources/EditorWindow/Components/InfoLinks.uxml?fileID=9197481963319205126&amp;guid=83354f036ef846d4a9cd24b4a6fff4a7&amp;type=3#InfoLinks" />
+    <ui:VisualElement name="HelpAndDocsTab" class="TabMenu" style="display: flex;">
+        <ui:GroupBox>
+            <ui:Label tabindex="-1" text="Help &amp; Documentation" display-tooltip-when-elided="true" name="LabelHelpTitle" style="width: 832px; height: 32px; background-color: rgba(118, 118, 118, 0); color: rgb(210, 210, 210); font-size: 24%; right: auto;" />
```

Comment:
As mentioned on a separate PR, please separate these styles into the uss file. That will help make this file readable and maintainable.

Reply:
Please don't resolve comments until the comment's addressed with commits or a response

Reply:
Applied classes and removed inline styles